### ISSUE TITLE
`-w`/`--wait` flag no longer works

### ISSUE THREAD

**Issue by julien-f** [flathub/com.visualstudio.code#261]
Hi,

I noticed in recent versions this flag no longer works, ie the command returns immediately without waiting for the window to close:

```
> flatpak run --filesystem=/tmp com.visualstudio.code -nw
> # returns with the window still opened
```

This breaks the use of vscode as Git editor (for commit, interactive rebase, etc.)

I'm not sure if it's related to the FlatHub version or not, I could not find any related issues on either repositories.

Best regards,

**Comment by bytzo** [comment]
This issue seems to have been fixed by #300.

**Comment by julien-f** [comment]
Indeed, it appear to be working now :slightly_smiling_face:
